extensions/nuget/src/NugetPackage.tsx

```diff
@@ -0,0 +1,30 @@
+export interface PackageType {
+  name: string;
+}
+
+export interface Version {
+  version: string;
+  downloads: number;
+  "@id": string;
```

Comment:
what is the reason to define it as `@id` and not `id`?

Reply:
because the json structure of response from nuget use both '@id' and 'id', i use it for coherence, can it be done in better way?